hypertrace-metrics-generator/hypertrace-metrics-generator-api/src/main/avro/metric-identity.avdl

```diff
@@ -0,0 +1,7 @@
+@namespace("org.hypertrace.metrics.generator.api")
```

Comment:
> Why are we still adding Avro messages? 
Not aware of this, is that the plan?

In either case, I will check with @laxman-traceable If I can directly go with java based serialization as I don't need them outside.

Reply:
My understanding - could be wrong - was that we initially used avro over kafka because that's what it had support for, and proto everywhere else. Newer versions have equal levels of proto support, so rather than using two different serialization formats, unifying on just proto for all new messages would be more desirable.

Reply:
Talked with @laxman-traceable. And as per him, avro goes better with kafka overall, and as you said the newer versions have support for schema registry for proto too. But, as you said, in this metrics pipeline, proto is the main transport layer, so he was also suggesting to go with Proto for consistency. Changing it to proto.

Reply:
Done